### PR TITLE
[REVIEW]Update nlp notebook to new cudf str api

### PR DESCRIPTION
**Pull request by VibhuJawa**
Update nlp notebook to the new cudf str api and removes nvstrings and nvtext references.

### CODE REVIEW

File: blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb [hunk 1]
```diff
@@ -4,13 +4,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Gutenburg NLP Analysis:\n",
+    "# Gutenburg NLP Analysis using Rapids\n",
```
Comment (randerzander) [inline blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb:7]:
```suggestion
    "# Gutenburg NLP Analysis using RAPIDS\n",
```

File: blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb [hunk 2]
```diff
@@ -211,13 +225,13 @@
     "\n",
     "Overall, avoiding regex can be more efficient since the algorithm is complex due to the richness of its features.  \n",
     "\n",
-    "For cases like removing multiple `characters` or `stop words`, a `general regex` can be overkill and `nvtext` provides some alternative methods which make this computation much faster. \n",
+    "For cases like removing multiple `characters` or `stop words`, a `general regex` can be overkill and `cudf.str` provides some alternative methods which make this computation much faster. \n",
     "\n",
-    "In this workflow we use the following `nvtext` functions:\n",
-    "* `nvstrings.replace_multi`: To replace the punctuations with a blank space.\n",
-    "* `nvtext.replace_tokens`: To replace the tokens with a empty space.\n",
+    "In this workflow we use the following `cudf.series.str` functions:\n",
```
Comment (randerzander) [inline blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb:230]:
```suggestion
    "In this workflow we use the following `cudf.Series.str` functions:\n",
```

File: blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb [hunk 3]
```diff
@@ -211,13 +225,13 @@
...
+    "* `str_ser.str.translate`: (Allows passing dict to replace the punctuations with a blank space.\n",
```
Comment (randerzander) [inline blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb:231]:
```suggestion
    "* `str_ser.str.translate`: (Allows passing dict to replace multiple punctuation characters with blank spaces.\n",
```

File: blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb [hunk 4]
```diff
@@ -997,6 +986,23 @@
     "\n",
     "Don't want to wait for the next release to use upcoming features? You can download our nightly containers from [Dockerhub](https://hub.docker.com/r/rapidsai/rapidsai-nightly) or install via [Conda](https://anaconda.org/rapidsai-nightly) to stay at the tip of our development branch."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other Examples of NLP workflows:\n",
+    "- https://github.com/rapidsai/tpcx-bb/tree/master/tpcx_bb/queries/q17\n",
```
Comment (randerzander) [inline blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb:995]:
```suggestion
    "- [tpcx-bb query 17](https://github.com/rapidsai/tpcx-bb/tree/master/tpcx_bb/queries/q17)\n",
```

Comment (randerzander) [reply blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb:995]: Same for the other query impl links

Comment (VibhuJawa) [reply blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb]: Should be resolved by: https://github.com/rapidsai/notebooks-contrib/pull/300/commits/cf149e05c2236ba00c399dc9a8c136eac49f7f0f

**Pull request merged**